xarray/conventions.py

```diff
@@ -506,6 +526,31 @@ def stackable(dim):
                     new_vars[k].encoding["coordinates"] = coord_str
                     del var_attrs["coordinates"]
                     coord_names.update(var_coord_names)
+            for attr_name in CF_RELATED_DATA:
+                if attr_name in var_attrs:
```

Comment:
Should we raise a warning here saying that we've started parsing these attributes and the folowing variables ... have been set as coords. ?

I'm thinking about backwards compatibility...

Reply:
I added a warning about variables moving from `data_vars` to `coords`

Reply:
I just changed the `stacklevel` so the warning would point to the `xarray.open_dataset` call, not the `cf_decode` call in `decode_cf`.